x-pack/dockerlogbeat/pipelinemanager/config.go

```diff
@@ -0,0 +1,75 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package pipelinemanager
+
+import (
+	"strings"
+
+	"github.com/pkg/errors"
+
+	"github.com/elastic/beats/v7/libbeat/common"
+	"github.com/elastic/beats/v7/libbeat/common/transform/typeconv"
+)
+
+// ContainerOutputConfig has all the options we'll expect from --log-opts
+type ContainerOutputConfig struct {
+	Endpoint    []string `struct:"output.elasticsearch.hosts,omitempty"`
```

Comment:
discuss naming: I wonder if we want to name this setting `host(s)` or `elasticsearch` :)

Reply:
Yah, `hosts` might be more consistent.